nats-server/configuration/gateways/README.md

```diff
@@ -48,14 +52,17 @@ The number of connections required to join clusters using clustering vs. gateway
 | 5 | 105 | 30 |
 | 30 | 4005 | 180 |
 
-## Interest Propagation
+A cluster section is not needed for gateways, they work with single server as well. Yet, they start to be useful when participating cluster consist of more than one server and they reduce the number of connections.
 
-Gateways propagate interest using three different mechanisms:
+## Interest Propagation
+Messages from clients directly connected to a gateway node will be sent along outgoing gateway connections according to the following three interest propagation mechanisms:
 
 * Optimistic Mode
 * Interest-only Mode
 * Queue Subscriptions
 
+Local interest permitting, the receiving gateway node sends the messages directly to it's subscribing clients as well as server within the cluster that have subscribing clients.
```

Comment:
...to it's subscribing clients as well as <any><all><a> server within the cluster<s> that have ...

Reply:
removed